src/swarm/neo/node/RequestSet.d

```diff
@@ -217,22 +225,49 @@ class RequestSet
 
         /***********************************************************************
 
-            Resumes the fiber or starts it if it is new or terminated.
+            Resumes the fiber or starts it if it is new or terminated. If
+            resuming (not starting) the fiber, `msg` is passed to the waiting
+            `suspendFiber()` call.
+
+            This method is called when a message for this request arrives. The
+            base class implements the basic behaviour and resumes the fiber.
+            However, in the node there is a case that needs special handing: the
+            message that starts the request.
+            When this initial message is received then the new `Request` object,
+            a subclass of `RequestOnConnBase`, is either created or fetched from
+            the object pool, and its `setReceivedPayload` is called, which calls
+            `resumeFiber_`. Since this is a new request the fiber needs to be
+            started rather than resumed. During the lifetime of the request
```

Comment:
I don't quite get the last sentence.

Reply:
What I'm trying to say is: The super class method always resumes the fiber when a request message has been received. This override changes the behaviour only for the first request message where, instead of resuming, it starts the fiber. For subsequent incoming messages it resumes the fiber i.e. resorts to super class method behaviour.

Reply:
Maybe something like "_For further arriving messages_ during the lifetime of the request super class behaviour applies i.e. the fiber is resumed."?

Reply:
I see. Yeah, that sounds better.